Misc/NEWS.d/next/Library/2022-02-05-22-14-44.bpo-46644.P--1Cz.rst

```diff
@@ -0,0 +1 @@
+No longer require valid typeforms to be callable. This allows :data:`typing.Annotated` to wrap :data:`typing.ParamSpecArgs` and :data:`dataclasses.InitVar`. Patch by Gregory Beauregard.
```

Comment:
Each line should be under 80 chars. However since this is usually merged into the next release's changelog anyways I'm not sure if it matters.

Reply:
Yes, I think blurb takes care of that when it generates the full changelog.